Cython/Coverage.py

```diff
@@ -45,6 +46,35 @@ def _find_dep_file_path(main_file, file_path):
     return abs_path
 
 
+def _get_actual_filename(filepath):
+    """
+    on windows:
+    search the file `filepath' on the file system 
+    and return it with the correct upper/lowercases
+    on linux: return the filepath unmodified
+    
+    Parameters
+    ----------
+    filepath : str
+    
+    Returns
+    -------
+    filepath : str
+    """
+    if not sys.platform.startswith('win'):
+        return filepath
+    dirs = filepath.split('\\')
+    # disk letter
+    test_name = [dirs[0].upper()]
+    for d in dirs[1:]:
+        test_name += ["%s[%s]" % (d[:-1], d[-1])]
```

Comment:
How to append to a list? Easy! Create a new one and concatenate it to the existing one! :D
(I know, it's now your code...)

Reply:
Thats whats happes if you copy&paste random code ;-)
I fixed this